packages/lb-components/src/components/longText/index.tsx

```diff
@@ -31,23 +32,52 @@ const LongText = (props: IProps) => {
     style,
     isToolTips,
     wordCount,
+    overflowMaxLines,
   } = props;
 
   const el = useRef<null | HTMLDivElement>(null);
   useSize(el);
   const isOverflow = el.current && el.current?.clientWidth < el.current?.scrollWidth;
 
-  if (wordCount) {
-    if (text?.length > wordCount) {
-      const showText = text.slice(0, wordCount);
+  if (wordCount || overflowMaxLines) {
+    const clampStyles = overflowMaxLines
+      ? {
+          display: '-webkit-box',
+          WebkitLineClamp: overflowMaxLines,
+          WebkitBoxOrient: 'vertical' as 'vertical',
+          overflow: 'hidden',
+        }
+      : {};
+    const TextDom = (
```

Comment:
Writing logic in hooks is not elegant and difficult to reuse. Try splitting (refactoring into components)